templates/base.html

```diff
@@ -7,130 +7,146 @@
 
 <!DOCTYPE html>
 <html lang="en">
-    <head>
```

Comment:
@nkarmi @waseem-omar @awartani please fix this file diff. because there was a change in indenting, far more lines appear changed than actually were. Please reset the indenting on this file so we can see just the important changes.